packages/edit-site/src/components/post-edit/index.js

```diff
@@ -159,7 +158,13 @@ function PostEditForm( { postType, postId } ) {
 
 	return (
 		<VStack spacing={ 4 }>
-			<PostEditHeader postType={ postType } postId={ postId } />
+			<PostCardPanel
+				postType={ postType }
+				postId={
+					ids.length === 1 ? parseInt( ids[ 0 ], 10 ) : undefined
+				}
+				postIds={ ids.length > 1 ? ids : undefined }
```

Comment:
Can this be just `postId` prop and accept an array or a scalar there?

Reply:
Yeah that works for me, I chose `postIds` for clarity, but for consumers a single `prop` would be easier.